rosbag2_transport/rosbag2_transport/__init__.py

```diff
@@ -28,5 +33,4 @@ def _import(name):
                          '#import-failing-even-with-library-present-on-the-system')
         raise
 
-
```

Comment:
This seems unrelated and the linter might complain.

Reply:
Fixed 3447ca4